mobile_app_connector/controllers/mobile_app_controller.py

```diff
@@ -14,6 +14,7 @@
 from werkzeug.exceptions import NotFound, MethodNotAllowed
 
 
+@http.route('/mobile-app-api')
```

Comment:
I never saw routes at the class-level, what is that for?

Reply:
most frameworks let you specify routes at both the class and method level (they get concatenated) so I was curious to see if it worked ;) will get rid of it